internal/manifests/collector/container_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v3"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
is this the same import as the one above it?

Reply:
it is. fixed.